### PR TITLE
EOS-11679: fs_ns_scan_cb() should be enhanced

### PR DESCRIPTION
**Pull request by gshipra**
Problem Statement :
EOS-11679: fs_ns_scan_cb() should be enhanced
Problem Description:
Code for fs_ns_scan_cb needs to be enhanced as suggested in defect.
Solution Overview:
Update log_write to have line number information.
Unit Test Cases:
UT passes

Signed-off-by: Shipra <shipra.gupta@seagate.com>

### CODE REVIEW

**Review by Anilsindhi — APPROVED** [review]
LGTM +1

File: c-utils/src/include/common/log.h [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 /* @todo : Improve this simplistic implementation of logging. */
 #define LOG(level, fmt, ...) \
-    log_write(level, "%s: " fmt "\n", __func__, ##__VA_ARGS__)
+    log_write(level, "%s: %d" fmt "\n", __func__, __LINE__, ##__VA_ARGS__)
```
Comment (ivan-alekhin) [inline c-utils/src/include/common/log.h:29]: What do you think about putting the space after the number?
```
// original:
     log_write(level, "%s: %d" fmt "\n", __func__, __LINE__, ##__VA_ARGS__) 
=>  foo: 123MessageText
// proposed change:
     log_write(level, "%s:%d " fmt "\n", __func__, __LINE__, ##__VA_ARGS__) 
=> foo:123 MessageText
```

**Review by pratyush-seagate — APPROVED** [review]
LGTM, +1

File: c-utils/src/include/common/log.h (hunk 1 above)
Comment (gshipra) [reply c-utils/src/include/common/log.h:29]: Done.

**Review by s-arya — APPROVED** [review]

**Pull request merged**